RELEASING/email_templates/result_pmc.j2

```diff
@@ -20,15 +20,15 @@ To: {{ receiver_email }}
 
 Subject: [RESULT] [VOTE] Release Apache {{ project_name }} {{ version }} based on Superset {{ version_rc }}
 
-Thanks to everyone that participated. The vote to release
+Thanks to everyone who participated. The vote to release
 Apache {{ project_name }} version {{ version }} based on {{ version_rc }} is now closed.
 
 {% if vote_negatives|length > 0 -%}
 The vote did NOT PASS with {{vote_bindings|length}} binding +1, {{ vote_nonbindings|length}} non-binding +1, and {{vote_negatives|length}} -1 votes:
 {% elif vote_bindings|length > 2 -%}
 The vote PASSED with {{vote_bindings|length}} binding +1, {{ vote_nonbindings|length}} non-binding +1, and {{vote_negatives|length}} -1 votes:
 {% else -%}
-The vote is non conclusive with {{vote_bindings|length}} binding +1, {{ vote_nonbindings|length}} non-binding -1, and {{vote_negatives|length}} -1 votes:
+The vote is non conclusive with {{vote_bindings|length}} binding +1, {{ vote_nonbindings|length}} non-binding +1, and {{vote_negatives|length}} -1 votes:
```

Comment:
Nice catches. I just wonder if vote_nonbinding is either + OR -? I'm not seeing where this is tabulated to be sure. I'm guessing you're right though :D

Reply:
It's defined [here](https://github.com/apache/superset/blob/8657fa753bbc20b6ff4159424d7385707156a349/RELEASING/generate_email.py#L104).

> A List of people with +1 non binding vote (ex: Ville)